### DISCUSSION TITLE
WDIO Allure & Video reports not working as expected in CI/CD Pipeline

### DISCUSSION THREAD

**Discussion by Subbu-raman** [webdriverio/webdriverio#7955] [Q&A]
Hi Everyone. We are using WDIO with Cucumber framework and we use Video and Allure reporters. We are currently experiencing two issues with Allure & video reports when running the WDIO tests in a pipeline (tried in both Bitbucket and Jenkins same issue occurs.) but the allure reporting works as expected in local. The issue was when the test step fails its not showing the video in the allure report albeit the video is generated and available in allure-results folder for the failed step and we use cucumber retry which is showing as a separate scenario in allure report instead of showing it under retry tab which skews the overall test count. Can you please let me know if anyone has faced similar issues when using allure reports in CI/CD pipeline. Below are my configs, code 
```
"@wdio/allure-reporter": "^7.16.14",
"@wdio/cucumber-framework": "^7.9.0",
"allure-commandline": "^2.17.2",
"@wdio/cli": "7.16.13",
"wdio-video-reporter": "^3.2.0",
```

Below is the WDIO.conf.ts for reports
```
reporters: [
        'spec',
        [video, {
            saveAllVideos: false,       // If true, also saves videos for successful test cases
            videoSlowdownMultiplier: 3, // Higher to get slower videos, lower for faster videos [Value 1-100]
            videoRenderTimeout: 5,
        }],
        ['allure', {
            outputDir: 'allure-results',
            disableWebdriverStepsReporting: true,
            disableWebdriverScreenshotsReporting: true,
            useCucumberStepReporter: true,
        }],
        ['cucumberjs-json', {
            jsonFolder: 'xray-json',
            language: 'en',
        },
        ],
    ],
```
The above mentioned issue only occurs for failed test in pipeline and works fine for passed test. I can see the video of the passed test in allure report in pipeline when the "saveAllVideos" is set to true.

**Comment by umarkhanTR** [comment]
seems wdio video reporter has issues running in Jenkins.

 'spec',
    [
      video,
      {
        saveAllVideos: true, // If true, also saves videos for successful test cases
        videoSlowdownMultiplier: 3, // Higher to get slower videos, lower for faster videos [Value 1-100]
        videoRenderTimeout: 5, // Max seconds to wait for a video to finish rendering
        videoFormat: 'webm',
      },
    ],
    [
      'allure',
      {
        outputDir: 'allure-results',
        disableWebdriverStepsReporting: true,
        disableWebdriverScreenshotsReporting: false,
        addConsoleLogs: true,
        disableMochaHooks: true,
      },
    ],
